.github/dependabot.yml

```diff
@@ -48,8 +34,13 @@ updates:
       # Addon framework, this shouldn't be upgraded on release branches
       - dependency-name: open-cluster-management.io/addon-framework
       - dependency-name: open-cluster-management.io/api
+      # These are included with submariner-operator
+      - dependency-name: github.com/submariner-io/admiral
+      - dependency-name: github.com/submariner-io/submariner
       # 2.6 tracks Submariner 0.13
-      - dependency-name: github.com/submariner-io/*
+      - dependency-name: github.com/submariner-io/submariner-operator
+        versions: ">= 0.14.0-m0"
+      - dependency-name: github.com/submariner-io/cloud-prepare
```

Comment:
Do you want to amend the commit message accordingly?

Reply:
sure